src/test/resources/fixtures/beaconSearch.json

```diff
@@ -0,0 +1,49 @@
+{
+  "meta": {
+    "count": 1,
+    "pageSize": 1
+  },
+  "data": [
+    {
+      "type": "beacon",
+      "id": "97b306aa-cbd0-4f09-aa24-2d876b983efb",
+      "attributes": {
+        "manufacturer": "Ocean Signal",
+        "model": "EPIRB1",
+        "manufacturerSerialNumber": "1407312904",
+        "chkCode": "9480B",
+        "batteryExpiryDate": "2020-02-01T00:00",
+        "lastServicedDate": "2020-02-01T00:00",
+        "uses": [
+          {
+            "environment": "MARITIME",
+            "activity": "SAILING",
+            "moreDetails": "More details of this vessel"
+          }
+        ],
```

Comment:
I don't know if this is per the API spec as I haven't used it previously, @NyWeb should be able to advise, but looking at the docs I would have expected the uses/owners/emergency contacts to be in [the relationships key of the JSON object](https://jsonapi.org/format/#document-resource-object-relationships).

I'm not sure if the above JSON is compliant with the spec with us having the relations as top level fields.  That being said, we do not have the API endpoints there anyway to retrieve uses/owner/emergency contacts by beacon id.

What are your thoughts on this - happy to jump on a call to discuss!

Reply:
lets catch up on this

Reply:
Concluded we should pick this up with the team before we progress further down this path